amethyst_renderer/src/renderer.rs

```diff
@@ -102,6 +104,15 @@ impl Renderer {
         #[cfg(feature = "opengl")]
         use glutin::GlContext;
 
+        let size = self.window()
+            .get_inner_size_pixels()
+            .expect("Unable to fetch window size, as the window went away!");
```

Comment:
Funny message, but if you could add "(window context lost, was the window closed?)" after, it might help the game devs figure out what happened more easily.